lib/active_merchant/billing/gateways/cecabank.rb

```diff
@@ -1,8 +1,8 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class CecabankGateway < Gateway
-      self.test_url = 'http://tpv.ceca.es:8000'
-      self.live_url = 'https://pgw.ceca.es'
+      self.test_url = 'https://tpv.ceca.es/tpvweb/'
+      self.live_url = 'https://pgw.ceca.es/tpvweb/'
```

Comment:
Just a heads up - I think we'll need to update the docs page (https://docs.spreedly.com/payment-gateways/cecabank/) with this new production URL.

Reply:
@jasonxp I'm confused about what exactly you think needs to be updated in the documentation.

The CECAbank documentation currently states that the API endpoint URL is `https://pgw.ceca.es`: 
![image](https://user-images.githubusercontent.com/42787645/61330148-008f7480-a7ed-11e9-95f8-11c1df9dfcbc.png)

The code above states that the production endpoint URL is still 'https://pgw.ceca.es'. I didn't think that we included paths as part of the documented production URL, especially considering that the previous code still relied on paths to get to the appropriate URL but were not explicitly documented as the endpoint URL:
https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/cecabank.rb#L171

Reply:
From a cursory look, I saw the documented production URLs matching the live URLs in ActiveMerchant. For example, Adyen includes the path in the docs: https://docs.spreedly.com/payment-gateways/adyen/. @curiousepic might have some insight since he worked on that gateway recently.

Reply:
I see what you're saying now. Perhaps it makes more sense to instead move the `/tpvweb/` part of the URL down to line 171 where the rest of the path gets added to the URL?

Reply:
I think it's fine either way in Active Merchant. Whether having the path would be beneficial in the documentation depends mostly on how customers use that piece of information. Feel free to :shipit:, as we could always tweak the docs later if we find out that would be useful.

Reply:
There are some cases where we would want more than the root to show up in our Docs, such as when there might be versioning via url path. That means an issue in Docs rather than here though, I think.